mesos-client/src/main/scala/mesosphere/mesos/conf/MesosConf.scala

```diff
@@ -0,0 +1,48 @@
+package mesosphere.mesos.conf
+
+import org.rogach.scallop.ScallopConf
+
+/**
+  * Configuring mesos v1 client
+  */
+class MesosConf(args: Seq[String]) extends ScallopConf(args) {
```

Comment:
Perhaps this config should be exposed via a config object, so the user of the client can expose CLI arguments as such?

Reply:
How about [Config](https://github.com/lightbend/config) from Typesafe/Lightbend?

Reply:
What is the decision on this?

Reply:
I'm not sure what the fascination with Typesafe conf is - my idea was that we can mix `MesosConf` with `MarathonConf` similar to how we do it now. But I understand that it makes it harder to use this client separately. 
I made it a simple case class with the option to move to Typesafe/whatever later.